src/Go/effects/netscriptGoImplementation.ts

```diff
@@ -337,28 +337,25 @@ export async function determineCheatSuccess(
 ): Promise<Play> {
   const state = Go.currentGame;
   const rng = new WHRNG(Player.totalPlaytime);
+  state.cheatCount++;
+  state.passCount = 0;
+
   // If cheat is successful, run callback
   if ((successRngOverride ?? rng.random()) <= cheatSuccessChance(state.cheatCount)) {
     callback();
-    state.cheatCount++;
     GoEvents.emit();
     return makeAIMove(state);
   }
   // If there have been prior cheat attempts, and the cheat fails, there is a 10% chance of instantly losing
   else if (state.cheatCount && (ejectRngOverride ?? rng.random()) < 0.1) {
```

Comment:
The `state.cheatCount` check here will now always be true. Feels like a bug?

Reply:
good catch, yeah